tools/deploy-api.ps1

```diff
@@ -0,0 +1,255 @@
+<#
+	.SYNOPSIS
+	Creates the file system folder structure along with IIS website and application for the given API.
+	.DESCRIPTION
+	Creates the file system folder structure for an API.
+	Creates the IIS website for the API.
+	Creates the application for the specified version of the API.
+	.PARAMETER site_name
+	The site name for the API. Required.
+	.PARAMETER app_name
+	The application name for the API. Required.
+	.PARAMETER app_version
+	The version of the API for the application. Required.
+	.PARAMETER site_hostname
+	The hostname of te API website. Required.

```

Comment:
`te` > `the`

Reply:
OK, this one I'll fix. 🙂

Reply:
It shows as a new file in this PR for whatever reason.